eraser.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containerd/containerd"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+func get_containerd_images() error {
```

Comment:
Go convention is to use mixed caps: https://golang.org/doc/effective_go#mixed-caps